platform/core/README.md

```diff
@@ -26,6 +26,9 @@ make ios_bindings
 
 - Building android core:
 
+Prerequisites:
+- You should install NDK(it is recommended to use Android Studio). A version >= r23 is required.
```

Comment:
Nit: space before parentheses. Maybe rephrase: "it's recommended anyway when you use Android Studio"